src/pages/home/home.ts

```diff
@@ -71,13 +72,17 @@ export class HomePage {
   isOnWater: boolean;
   isInGroup: boolean;
 
+  //Will show the group test service on dismiss of group.
+  showGroupTestButton: boolean = false;
```

Comment:
Since you are already assigning `showGroupTestButton` to `false`, typescript can already infer that it is of type boolean. Therefore, there is no need to explicitly write the type.

Reply:
Also `showGroupTestButton` is not being used in the HTML... If that was the intention for this.